husky_gazebo/launch/husky_playpen.launch

```diff
@@ -25,8 +25,8 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
 -->
 <launch>
 
-  <arg name="laser_enabled" default="true"/>
-  <arg name="realsense_enabled" default="false"/>
+  <arg name="laser_enabled" default="$(optenv HUSKY_LASER_LMS1XX_ENABLED 0)"/>
```

Comment:
Why change the simulation args to envars?  Seems a lot easier to have them as args in that given use-case.

Reply:
My thinking on this was a few disparate points:

1. Not every arg is exposed in e.g. the simulation launch files.  Ensuring everything is configurable with envars means everything can be set up in one sourced bash script, eliminating the need to use command-line args
2. This just affects the default value; users are still welcome to use the command-line parameters if they want.
3. All of our other simulations launch the base, naked robot by default.  Having Husky's simulations add a primary laser by default seemed inconsistent, which is why I changed the default value to false.
4. If we're wanting to keep the simulation close to the actual robot, it makes sense that if you copy the robot's `/etc/ros/setup.bash` file into your sim environment, the robot should have the same sensors.  But if we default the laser to `true`, as it was before, then you need to modify the setup file to explicitly disable the laser for the simulation or are forced to set the command-line parameter when launching the sim.

If we wanted to preserve the old behaviour and keep the laser on by default I think the best way would be to change `optenv HUSKY_LMS1XX 0` to `optenv HUSKY_LMS1XX 1`. That keeps most of the envar support intact, but it would keep the simulation & physical robots' setup.bash files inconsistent with each other, like I mentioned in point 4 above.

Reply:
That is fair enough, keeping the default behavior would be ideal since many tutorials assume that the laser is there when simulating a Husky. Also, the tutorial already has the envar in it [here](http://www.clearpathrobotics.com/assets/guides/melodic/husky/additional_sim_worlds.html#customizing-husky-s-payload). The Ridgeback simulation has a laser by default as well.  Regarding simulation, `velodyne_gazebo_plugins` is missing for the velodyne simulation.

Reply:
That particular simulation is just the extra worlds Dave made.  The husky_playpen sim previously ignored the HUSKY_LMS1XX envar completely.

Anyway, I've changed the default for the simulation launch files to be 1; you can use an envar or command-line arg to specifically disable it if needed, but this should add the extra functionality while preserving the legacy behaviour.

Also added the missing dependency to the husky_gazebo package.

Reply:
Looks great, thanks Chris!